src/NUnitFramework/tests/Internal/EventListenerTextWriterTests.cs

```diff
@@ -95,7 +97,7 @@ public void TestWriteStringArg2()
             ListenerWriter.Write(format, arg0, arg1);
             ListenerWriter.WriteLine(format, arg0, arg1);
 
-            var expected = "05.00 @";
+            var expected = $"05{decimalSeparator}00 @";
```

Comment:
If this is supposed to be CurrentCulture and not InvariantCulture, an we do `$"{5:00.00} @"` for these instead?